build-tools/packages/build-cli/src/config.ts

```diff
@@ -378,3 +374,8 @@ export function getDefaultInterdependencyRange(
 
 	return interdependencyRange ?? DEFAULT_INTERDEPENDENCY_RANGE;
 }
+
+/**
+ * The default interdependency range we use when one is not provided.
+ */
+const DEFAULT_INTERDEPENDENCY_RANGE: InterdependencyRange = "workspace:~";
```

Comment:
This doesn't seem like the right fix. The only reason that this value should be used is if the values from the fluidBuild config could not be loaded. That seems like the core bug - what is causing the function above, getDefault InterdependencyRange, to return `DEFAULT_INTERDEPENDENCY_RANGE`?

Reply:
Ahhh, I think I understand the root cause now... The changes made in #21967 weren't fully back-compat. The config for each release group's bump behavior is now missing... the flub config should have a `bump` section.